src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx

```diff
@@ -111,7 +111,16 @@ export const ConversationsList: React.FC<{
               index={index}
               dataUieName="item-conversation"
               conversation={conversation}
-              onClick={createNavigate(generateConversationUrl(conversation.id, conversation.domain))}
+              onClick={event => {
+                if (event.type === 'keydown') {
+                  createNavigateKeyboard(
+                    generateConversationUrl(conversation.id, conversation.domain),
+                    true,
+                  )(event as unknown as React.KeyboardEvent);
+                } else {
+                  createNavigate(generateConversationUrl(conversation.id, conversation.domain))(event);
+                }
+              }}
```

Comment:
My suggestion:
```ts
const handleClick = (event: React.MouseEvent<HTMLDivElement, MouseEvent> | React.KeyboardEvent<HTMLDivElement>) => {
    if ('key' in event) {
      createNavigateKeyboard(generateConversationUrl(conversation.id, conversation.domain), true)(event);
    } else {
      createNavigate(generateConversationUrl(conversation.id, conversation.domain))(event);
    }
  };
```
We allow to pass both mouse and keyboard events, `'key' in event` will narrow the type to keyboard, otherwise it has to be mouseevent (the only one left from the union.